customer_service/lib/FileManager/File/PersistFileHandler.php

```diff
@@ -0,0 +1,233 @@
+<?php
+namespace Lib\FileManager\File;
+
+use Lib\Database\DatabaseManager;
+use Lib\ErrorReporter\ErrorReporter;
+use Lib\ErrorReporter\ErrorNodeFactory;
+
+class PersistFileHandler
+{
+    private $file;
+    private $files;
+    private $status;
+
+    public function __construct()
+    {
+    }
+
+    public function persistFile($sessionId, $fileGroupId, $fileType, $oldName, $fileDirPath, $extension, $size)
+    {
+        $error = "";
+        $response = "";
+        $errorNodeFactory = new ErrorNodeFactory();
+        $queryHolder="SET @sID=''";
+        $querySelectHolder="SELECT @sID AS 's_id'";
+        $query = "CALL prnInsFile('".$sessionId."', '"
+        .$fileGroupId."', '".$fileType."', '".$oldName
+        ."', '".$fileDirPath."', '".$extension."', '".$size."', @sID);";
+
+        if (DatabaseManager::mysql_query($queryHolder)) {
+            if (DatabaseManager::mysql_query($query)) {
+                $result = DatabaseManager::mysql_query($querySelectHolder);
+                if ($result) {
+                    $num_results = DatabaseManager::mysql_num_rows($result);
+                    if ($num_results>0) {
+                        $row = DatabaseManager::mysql_fetch_array($result);
+                        if (strcmp($row["s_id"], "")!=0) {
+                            $this->status = $row["s_id"];
+                        } else {
+                            $errorNode = $errorNodeFactory->createPersistenceError(
+                                "Class:FileHandler->persistFile; Empty status returned ",
+                                "Unable to register file.<br/>Please try again Later"
+                            );
+                            ErrorReporter::addNode($errorNode);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Avoid using static access to class '\Lib\ErrorReporter\ErrorReporter' in method 'persistFile'.](https://app.codacy.com/gh/BuildForSDG/Team-064-Backend/pullRequest?prid=5616370)